python-project-template/.pre-commit-config.yaml.jinja

```diff
@@ -150,11 +150,11 @@ repos:
                   # Note: -M requires next 3 args to be builder, source, output
             "html", # Specify builder
             "./docs", # Source directory of documents
-            "./docs/build/html", # Output directory for rendered documents
+            "./_readthedocs", # Output directory for rendered documents
             "-T", # Show full trace back on exception
             "-E", # Don't use saved env; always read all files
             "-d", # Flag for cached environment and doctrees
-            "./docs/build/doctrees", # Directory
+            "./docs/_build/doctrees", # Directory
             "-D", # Flag to override settings in conf.py
             "exclude_patterns=notebooks/*", # Exclude our notebooks from pre-commit
```

Comment:
this remains a difference. The pre-commit excludes notebooks while the sphinx builds include the notebooks.
I am only noting this since we really do not want the precommit to buid notebooks (dependency on pandocs).